batcher/aligned/src/main.rs

```diff
@@ -528,6 +558,20 @@ async fn main() -> Result<(), AlignedError> {
                 }
             }
         }
+        GetUserFirstNonce(args) => {
+            let address = H160::from_str(&args.address).unwrap();
+            let network = args.network.into();
+            match get_nonce_from_ethereum(&args.eth_rpc_url, address, network).await {
+                Ok(nonce) => {
+                    let first_nonce = nonce + 1;
+                    info!("Nonce for address {} is {}", address, first_nonce);
+                }
```

Comment:
I don't know if this is right. If the user has no proof submitted, we should not increment the value, maybe this request should be handled by batcher server.

Reply:
@uri-99 please confirm the convention. There will be a difference between what the user sees in the batcher and what is returned by this function (when a users proof has not been submitted on chain but is in the batcher queue).

Reply:
yes the batcher should handle it